metadata-ingestion/src/datahub/ingestion/source/usage/redshift_usage.py

```diff
@@ -182,18 +173,17 @@ def _make_usage_query(self, query: str) -> str:
 
     def _make_redshift_operation_aspect_query(self, table_name: str) -> str:
         return f"""
-        SELECT DISTINCT ss.userid,
-               ss.query,
-               ss.rows,
-               sui.usename,
-               ss.tbl,
-               sq.querytxt,
-               sti.database,
-               sti.schema,
-               sti.table,
-               sq.starttime,
-               sq.endtime,
-               sq.aborted
+        SELECT DISTINCT ss.userid as userid,
```

Comment:
We could replace the whole logic of getting insert and delete types separately with one union query like this:
```
SELECT DISTINCT ss.userid as userid,
       ss.query as query,
       ss.rows as rows,
       sui.usename as username,
       ss.tbl as tbl,
       sq.querytxt as querytxt,
       sti.database as database,
       sti.schema as schema,
       sti.table as table,
       sq.starttime as starttime,
       sq.endtime as endtime,
       'insert' as type
FROM stl_insert ss
JOIN svv_table_info sti ON ss.tbl = sti.table_id
JOIN stl_query sq ON ss.query = sq.query
JOIN svl_user_info sui ON sq.userid = sui.usesysid
AND ss.rows > 0
AND sq.aborted = 0
union
SELECT DISTINCT ss.userid as userid,
       ss.query as query,
       ss.rows as rows,
       sui.usename as username,
       ss.tbl as tbl,
       sq.querytxt as querytxt,
       sti.database as database,
       sti.schema as schema,
       sti.table as table,
       sq.starttime as starttime,
       sq.endtime as endtime,
       'delete' as type
FROM stl_delete ss
JOIN svv_table_info sti ON ss.tbl = sti.table_id
JOIN stl_query sq ON ss.query = sq.query
JOIN svl_user_info sui ON sq.userid = sui.usesysid
AND ss.rows > 0
AND sq.aborted = 0
ORDER by endtime desc
```

Reply:
This is a great idea! Will do it in  a separate PR.